### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by gcotelli**

### CODE REVIEW

File: docs/ExceptionHandling.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Exception handling
+
+- `on:except:do:` provides an extension to the exception handling machinery to allow handling an exception, but ignoring some subclass of it. For example:
```
Comment (fortizpenaloza) [inline docs/ExceptionHandling.md]: ...provides an extension to the exception handling allowing to ignore some subclass of the exception being handled. For example:

File: docs/Installation.md [hunk 2]
```diff
@@ -36,3 +36,12 @@ baseline: spec
...
 ```
+
+## Provided groups
+
+- `Deployment` will load all the packages needed in a deployed application
+- `Tests` will load the test cases
+- `Dependent-SUnit-Extensions` will load the extensions to the SUnit framework
+- `Tools` will load the extensions to the SUnit framework and development tools (inspector and spotter extensions)
+- `CI` is the group loaded in the continuous integration setup
+- `Development` will load all the needed packages to develop the project
```
Comment (fortizpenaloza) [inline docs/Installation.md]: ...to contribute to the project.

File: docs/Installation.md [hunk 3]
```diff
@@ -36,3 +36,12 @@ baseline: spec
 		do: [ self setUpDependencies: spec.
 			spec package: 'My-Package' with: [ spec requires: #('Buoy') ] ]
 ```
+
+## Provided groups
+
+- `Deployment` will load all the packages needed in a deployed application
```
Comment (fortizpenaloza) [inline docs/Installation.md:42]: `Deployment` will load all the packages needed in a deployed application. Aimed to be used when setting Buoy as a dependency.

What do you think? I'm starting to believe that we should rename this group in all of the @ba-st projects.

**Review by fortizpenaloza — COMMENTED** [review]
My 2 cents

File: docs/Installation.md (hunk 3 above)
Comment (gcotelli) [reply docs/Installation.md:42]: It's the group you want to depend on for the deployment of your app, but not the only dependency you can have on Buoy.. for the development of your app I would include the Tools group for example. I'm happy with the name.

Comment (fortizpenaloza) [reply docs/Installation.md:42]: Ok, good point. Thanks!

**Review by jvanecek — APPROVED** [review]

**Pull request merged**